editor/script/paint.js

```diff
@@ -361,13 +382,13 @@ function PaintTool(canvas, roomTool) {
 		tileIndex = Object.keys(tile).length - 1;
 	}
 
-	function newSprite(id) {
+	function newSprite(id, imageData) {
 		if (id)
 			self.drawing.id = id; //this optional parameter lets me override the default next id
```

Comment:
You know I don't think this optional parameter is ever actually used - do you mind removing it from these `newX` functions? That way you don't have to pass `undefined` into them for the id parameter